packages/react-recommendations/src/DefaultChildren.tsx

```diff
@@ -1,22 +1,24 @@
-import React from 'react';
+import React, { createElement, Fragment } from 'react';
 
 import { ChildrenProps } from './types';
 import { cx } from './utils';
 
 export function DefaultChildren<TObject>(props: ChildrenProps<TObject>) {
   if (props.recommendations.length === 0) {
-    return <props.Fallback />;
+    return <props.Fallback createElement={createElement} Fragment={Fragment} />;
```

Comment:
We might want to pass `createElement` and `Fragment` in the `FBT` and `RP` components directly, otherwise they're not bound to these renderer props anywhere else than in our `DefaultChildren` component.

https://github.com/algolia/ui-components/blob/cf5586617855048d09cfe414ede458ddeace92aa/packages/react-recommendations/src/FrequentlyBoughtTogether.tsx#L40-L50

The same applies with `Header` and `View`.

Wdyt?

Reply:
Good idea! Is that what you had in mind? https://github.com/algolia/ui-components/pull/26/commits/fac42ac4ebd3e53cd2839862f37c052658017376